### PR TITLE
Implement continuous testing of server and client code.

### PR DESCRIPTION
**Pull request by jschwartzentruber**
Current results are here:
[travis](https://codecov.io/gh/jschwartzentruber/FuzzManager/branch/srvtests)
[codecov](https://travis-ci.org/jschwartzentruber/FuzzManager)

### CODE REVIEW

File: FTB/Signatures/test_CrashInfo.py [hunk 1]
```diff
@@ -20,6 +21,8 @@
     NoCrashInfo, MinidumpCrashInfo, AppleCrashInfo, CDBCrashInfo, RustCrashInfo
 from FTB.Signatures.CrashSignature import CrashSignature
 
+CWD = os.path.dirname(__file__)
```
Comment (choller) [inline FTB/Signatures/test_CrashInfo.py]: Afaik, this needs to be `os.path.dirname(os.path.realpath(__file__))` instead to properly cope with relative paths and/or symlinks.

File: server/covmanager/SourceCodeProvider/test_sourcecodeprovider.py [hunk 2]
```diff
@@ -88,6 +88,7 @@ def runTest(self):
         parents = provider.getParents("c3abaa766d52")
         self.assertEqual(len(parents), 0)
 
+@unittest.skipIf(not os.path.isdir("/home/decoder/Mozilla/repos/mozilla-central-fm"), reason="not decoder")
```
Comment (choller) [inline server/covmanager/SourceCodeProvider/test_sourcecodeprovider.py]: ( ╯°□°)╯ ┻━━┻ I didn't even mean to check this in. But it's ok to disable it for now until we can properly write a test. And this is a really creative way to do so ;D

File: server/covmanager/views.py [hunk 3]
```diff
@@ -349,8 +349,7 @@ class CollectionViewSet(mixins.CreateModelMixin,
         CollectionFilterBackend
     ]
 
-class RepositoryViewSet(mixins.CreateModelMixin,
-                        mixins.ListModelMixin,
+class RepositoryViewSet(mixins.ListModelMixin,
```
Comment (choller) [inline server/covmanager/views.py]: Nice, I guess this allowed creating repositories remotely before your change?

File: FTB/Signatures/test_CrashInfo.py (hunk 1 above)
Comment (jschwartzentruber) [reply FTB/Signatures/test_CrashInfo.py]: Possibly, yeah. I'll merge and then push this fix to master.

File: server/covmanager/views.py (hunk 3 above)
Comment (jschwartzentruber) [reply server/covmanager/views.py]: Yeah.. It isn't a terrible thing but I didn't think it was intended.

**Pull request merged**